bitnami/rabbitmq/values.yaml

```diff
@@ -448,6 +448,12 @@ advancedConfiguration: |-
 ##
 advancedConfigurationExistingSecret: ""
 
+## This subsystem was introduced in RabbitMQ 3.8.0 to allow rolling upgrades of cluster members without shutting down the entire cluster.
+## Feature flags are a mechanism that controls what features are considered to be enabled or available on all cluster nodes. If a feature flag is enabled, so is its associated feature (or behavior). If not then all nodes in the cluster will disable the feature (behavior).
+## e.g, drop_unroutable_metric,empty_basic_get_metric,implicit_default_bindings,maintenance_mode_status,quorum_queue,virtual_host_metadata
+# @param featureFlags that controls what features are considered to be enabled or available on all cluster nodes.
```

Comment:
I'm seeing the following error in the `readme-generator-for-helm` action step:

```
######
The following errors must be fixed before proceeding
######

ERROR: Missing metadata for key: featureFlags
```

Could you check it? I believe it shoudl be like this:

```suggestion
## @param featureFlags that controls what features are considered to be enabled or available on all cluster nodes.
##
```

Reply:
done